pkgs/development/python-modules/avro-python3/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "avro-python3";
+  version = "1.8.2";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "f82cf0d66189600b1e6b442f650ad5aca6c189576723dcbf6f9ce096eab81bd6";
+  };
```

Comment:
looks like the source distribution doesn't contain tests, it would be prefereable to check out from git hub. E.g fetchFromGitHub

Reply:
If we try to fetch from github, then lot of other languages(like java, c, c++, ...) code is coming, which is not relevant.